app/bundles/FormBundle/Model/SubmissionModel.php

```diff
@@ -410,6 +398,19 @@ public function saveSubmission($post, $server, Form $form, Request $request = nu
             return ['errors' => $validationErrors];
         }
 
+        // Create/update lead
+        if (!empty($leadFieldMatches)) {
+            $lead = $this->createLeadFromSubmit($form, $leadFieldMatches, $leadFields);
+        }
+
+        $trackedDevice = $this->deviceTrackingService->getTrackedDevice();
+        $trackingId    = ($trackedDevice === null ? null : $trackedDevice->getTrackingId());
+
+
```

Comment:
```suggestion
```
@anton-vlasenko the CS check fails probably because there are 2 empty rows here

Reply:
@escopecz Sorry, I saw your comment but then I lost it somewhere in my inbox and I wasn't able to find it.